libs/s25main/ingameWindows/iwStatistics.cpp

```diff
@@ -278,29 +279,62 @@ void iwStatistics::Draw_()
     DrawStatistic(currentView);
 }
 
-void iwStatistics::DrawPlayerBoxes()
+void iwStatistics::DrawPlayerOverlays()
 {
-    unsigned short startX = 126 - numPlayingPlayers * 17;
-    DrawPoint drawPt = GetDrawPos() + DrawPoint(startX, 68);
+    unsigned short startX = 126 - (numPlayingPlayers - 1) * 17;
+    DrawPoint drawPt = GetDrawPos() + DrawPoint(startX - 17, 45 - 23);
```

Comment:
Let's make this explicit here already, not copying that strange stuff:
```suggestion
    DrawPoint drawPt = GetDrawPos() + DrawPoint(126 - numPlayingPlayers * 17, 22);
```